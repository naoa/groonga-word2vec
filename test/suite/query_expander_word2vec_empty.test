register word2vec/word2vec

table_create table
column_create table text --type ShortText

select --table table --match_columns text --query database --query_expander QueryExpanderWord2vec

