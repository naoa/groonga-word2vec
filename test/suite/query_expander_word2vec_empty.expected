register word2vec/word2vec
[[0,0.0,0.0],true]
table_create table
[[0,0.0,0.0],true]
column_create table text --type ShortText
[[0,0.0,0.0],true]
select --table table --match_columns text --query database --query_expander QueryExpanderWord2vec
[[0,0.0,0.0],[[[0],[["_id","UInt32"],["text","ShortText"]]]]]
#|w| [plugin][word2vec][load] Input file not found : db/db_w2v.bin
